### DISCUSSION TITLE
Combining LayerStyle and Hover

### DISCUSSION THREAD

**Discussion by baumandm** [chakra-ui/chakra-ui#3273] [Q&A] [answered]
I'd like to apply a [Layer Style](https://chakra-ui.com/docs/features/text-and-layer-styles#layer-style) on hover, e.g. provide a box shadow.  I imagined it would look like this:

```js
      <Box _hover={{ layerStyle: 'selected' }}>
```

But that's not supported according to the type definitions.  Is this possible without unfurling the Layer Style contents into the `_hover` property?

**Comment by dodas** [comment, marked as answer]
Hi @baumandm,

[layerStyles/textStyles styles has been now moved to the `system`](https://github.com/chakra-ui/chakra-ui/pull/3245) and the exact code you posted will soon work.
We just need to wait for the next release that will come hopefully this friday (5.2.2020).

Another option (which works today) would be to hard-code the layerStyle's props into _hover, like this:
```ts
export const theme = extendTheme({
  layerStyles: {
    hoverSelected: {
      _hover: {
        color: "red.500"
      },
    },
  },
})

<Box layerStyle="hoverSelected">

**Reply by baumandm** [reply]
Perfect, thanks for the update!
